spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/configuration.adoc

```diff
@@ -377,31 +377,184 @@ Request to clean up resources for task execution 1 has been submitted
+This section covers customization of how scheduled tasks are configured. Scheduling of tasks is enabled by default in the Spring Cloud Data Flow Kubernetes Server. Properties are used to influence settings for scheduled tasks and can be configured on a global or per-schedule basis.
 
-* `SPRING_CLOUD_SCHEDULER_KUBERNETES_IMAGE_PULL_POLICY` - The Image Pull Policy to use when pulling Docker containers. Defaults to `IfNotPresent`.
-* `SPRING_CLOUD_SCHEDULER_KUBERNETES_RESTART_POLICY` - The Restart Policy to use when a container exits. Defaults to `Never`.
-* `SPRING_CLOUD_SCHEDULER_KUBERNETES_ENTRY_POINT_STYLE` - The Entry Point Style to determine how application properties are made available to the container. Defaults to `exec`.
-* `SPRING_CLOUD_DATAFLOW_FEATURES_SCHEDULES_ENABLED` - Enables or disables the Scheduling support. Defaults to `true`.
+NOTE: Unless noted, properties set on a per-schedule basis always take precedence over properties set as the server configuration. This arrangement allows for the ability to override global server level properties for a specific schedule.
 
-An example of customizing each of these in the server deployment YAML would be as follows:
+See https://github.com/spring-cloud/spring-cloud-scheduler-kubernetes/blob/master/src/main/java/org/springframework/cloud/scheduler/spi/kubernetes/KubernetesSchedulerProperties.java[`KubernetesSchedulerProperties`] for more on the supported options.
+
+=== Entry Point Style
+
+An Entry Point Style affects how application properties are passed to the task container to be deployed. Currently, three styles are supported:
+
+* `exec`: (default) Passes all application properties as command line arguments.
+* `shell`: Passes all application properties as environment variables.
+* `boot`: Creates an environment variable called `SPRING_APPLICATION_JSON` that contains a JSON representation of all application properties.
+
+You can configure the entry point style as follows:
+
+====
+[source,options=nowrap]
+----
+scheduler.kubernetes.entryPointStyle=<Entry Point Style>
+----
+====
+
+Replace `<Entry Point Style>` with your desired Entry Point Style.
+
+You can also configure the Entry Point Style at the server level in the container `env` section of a deployment YAML, as the following example shows:
 
 ====
 [source]
 ----
 env:
-- name: SPRING_CLOUD_SCHEDULER_KUBERNETES_IMAGE_PULL_POLICY
-  value: 'Always'
-- name: SPRING_CLOUD_SCHEDULER_KUBERNETES_RESTART_POLICY
-  value: 'OnFailure'
 - name: SPRING_CLOUD_SCHEDULER_KUBERNETES_ENTRY_POINT_STYLE
-  value: 'shell'
-- name: SPRING_CLOUD_DATAFLOW_FEATURES_SCHEDULES_ENABLED
-  value: 'false'
+  value: entryPointStyle
 ----
 ====
 
-See https://github.com/spring-cloud/spring-cloud-scheduler-kubernetes/blob/master/src/main/java/org/springframework/cloud/scheduler/spi/kubernetes/KubernetesSchedulerProperties.java[`KubernetesSchedulerProperties`] for more on the supported options.
+Replace `entryPointStye` with the desired Entry Point Style.
```

Comment:
entryPointStyle is misspelled.  Will fix on merge.

Reply:
will fix on push